conda-environment.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
   - jinja2 >=3.0,<3.1
   - metomi-isodatetime >=1!3.0.0, <1!3.1.0
   # Constrain protobuf version for compatible Scheduler-UIS comms across hosts
-  - protobuf >=3.19.0,<3.20.0
+  - protobuf >=4.21.2
```

Comment:
Are we ok to un-pin the protobuf version.

If cylc-flow uses protobuf `4.21` and the UIS uses `4.42` would that be a problem? If the UIS uses `5.10` would that be a problem?

Reply:
Maybe just do `>=4.21.1,<4.30.0`   (can't do `4.21.0` as I raised an issue with that one, and they fixed it)